apps/storefront/src/pages/Login/index.tsx

```diff
@@ -222,11 +222,19 @@ export default function Login(props: PageProps) {
 
     if (isCheckout) {
       try {
-        await loginCheckout(data);
-        window.location.href = CHECKOUT_URL;
+        const response = await loginCheckout(data);
+
+        if (response.status === 400 && response.type === 'reset_password_before_login') {
+          b2bLogger.error(response);
+          await getForcePasswordReset(data.emailAddress);
+        } else {
```

Comment:
I would change this to:
```suggestion
        if (!response.ok) {
          b2bLogger.error(response);
          if(response.type === 'reset_password_before_login') {
            await getForcePasswordReset(data.emailAddress);
          }
        } else {
```

Reply:
the problem with this is that there's more error types than that so I'd rather leave it like this - if a similar support ticket is raised where nothing happens we'll know that its a different case than this

Reply:
It makes sense